README.md

```diff
@@ -119,10 +119,13 @@ Commands to run the tests:
 
 ```js
 // Open Cypress Dashboard that provides insight into what happened when your tests ran
-yarn run cypress
+yarn run cy
```

Comment:
Why are these changes in this PR?

Reply:
While testing the impacts of this change, I saw that we changed that commands in `package.json` but didn't update the README file. Since I was already changing the code here, I thought it would be ok to fix the documentation

Reply:
It is a totally unrelated change, so it should be made in a separate commit and separate PR, with a specific commit message.

Reply:
Okay, I will remove from here and create another PR for this

Reply:
Done! Created the PR https://github.com/OHIF/Viewers/pull/610 to handle this changes